src/portable/portablectl.c

```diff
@@ -1185,6 +1283,15 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_no_block = true;
                         break;
 
+                case ARG_EXTENSION: {
+                        int r;
```

Comment:
"int r;" is a bit special for us, it's that one special variable in our coding style for storing function error codes in. It hence gets the special privilege to always be declared at the top-level scope of the function. Let's do that here too, move the "int r;" to the top of the function. This then allows you to get rid of the {}